packages/wonder-blocks-tokens/src/theme/primitive/font.ts

```diff
@@ -78,4 +78,8 @@ export const font = {
             xxlarge: sizing.size_400, // Title
         },
     },
+    textDecoration: {
+        offset: sizing.size_020,
+        thickness: sizing.size_010,
+    },
```

Comment:
It would be helpful to have a separate changeset for the `tokens` package for the newly added text decoration and border radius tokens so it gets included in the changelog 😄

Reply:
There is a changelog already for `wonder-blocks-tokens` - I can add a mention about the textDecoration stuff. Should it be a totally separate changeset entry?

Reply:
I think a separate one makes sense! 

The changeset description is going to show up in each of the packages changelog! So adding something like "Added new border radius and text decoration tokens" to the existing changeset file will get added to the `link` and `breadcrumbs` packages too, which is less relavent!

https://github.com/Khan/wonder-blocks/blob/2290ea4afd21e39dbda69a517d6ac3272c48afb0/.changeset/lazy-dolphins-stare.md?plain=1#L1-L7

I've been thinking more about our changeset descriptions and changelogs since I think they will probably get utilized more now that other teams (ie. assessments) will be managing their own WB dependencies later on.

Let me know if you have any thoughts on this!

Reply:
That totally makes sense. I separated the entries, thanks for suggesting it!